News/Controllers/ArticleController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using News.Entities.Models;
+using News.Services.ServicesInterface;
+
+namespace News.Controllers;
+
+[ApiController]
+[Route("article")]
+public class ArticleController : ControllerBase
+{    
+    private readonly IArticleService _service;
+    public ArticleController(DbContext context, IArticleService service)
+    {        
+        _service = service;
+    }
+
+    [HttpGet]
+    public IActionResult GetById([FromQuery] Guid id)
+    {
+        var result = _service.GetById(id);
+        if (result is null)
+        {
+            return NotFound();
+        }
+
+        return Ok(result);
+    }
+
+
+    [HttpPost]
+    public IActionResult Add([FromBody] ArticleModel model)
+    {
+        _service.Add(model);        
+
+        return Created();
+    }
+
+    [HttpPut]
+    public IActionResult UpdateFilm([FromBody] ArticleModel model)
+    {
+        _service.Update(model);       
+
+        return Accepted();
+    }
+
+    [HttpDelete]
+    public IActionResult DeleteFilmById(Guid id)
+    {
+        var result = _service.GetById(id);
```

Comment:
How do you get an id here? I guess you should get it also from route.

Reply:
What's the point of putting something in the route that you're going to delete?

Reply:
Because you need to get this ID. The same like you are getting ID in GetById

Reply:
Good, that makes sense!

Reply:
done